src/test/resources/test_data/sar_content/hardstopLicence.json

```diff
@@ -0,0 +1,89 @@
+{
+  "content": {
+    "licences": [
+      {
+        "kind": "HARD_STOP",
+        "id": 1,
+        "typeCode": "AP",
+        "version": "1.1",
+        "statusCode": "IN_PROGRESS",
+        "nomsId": "A1234AA",
+        "bookingId": 54321,
+        "appointmentPerson": null,
+        "appointmentTime": null,
+        "appointmentTimeType": null,
+        "appointmentAddress": null,
+        "appointmentContact": null,
+        "approvedDate": null,
+        "approvedByUsername": null,
+        "submittedDate": null,
+        "approvedByName": null,
+        "supersededDate": null,
+        "dateCreated": "27/07/2022 15:00:00",
+        "createdByUsername": "smills",
+        "dateLastUpdated": null,
+        "updatedByUsername": null,
+        "standardLicenceConditions": [
+          {
+            "code": "goodBehaviour",
+            "text": "Be of good behaviour"
+          },
+          {
+            "code": "notBreakLaw",
+            "text": "Do not break any law"
+          },
+          {
+            "code": "attendMeetings",
+            "text": "Attend meetings"
+          }
+        ],
+        "standardPssConditions": [],
+        "additionalLicenceConditions": [],
```

Comment:
shouldn't this have an additional condition fed through? its added by default through test data

Reply:
No I am passing emptyMap() for conditionSubmissionStatus

Reply:
Updated the test case to fetch additional conditions